wiki/osu!_wiki_Contribution_Guide/Common_Issues/en.md

```diff
@@ -70,66 +77,6 @@ You'll need to run a GitHub workflow written by osu! wiki contributors.
 
 ![GitHub Actions Workflow - Workflow Overview](img/github-actions-workflow-overview.png "GitHub Actions Workflow - Workflow Overview")
 
-#### Using the command line
```

Comment:
while I agree on the removal, I can't decide between two choices:

1. this needs to resurface in a separate article, because it's a valid method of keeping the branch in sync (including hard overwrite).
2. this doesn't need to be mentioned, because if someone's using command line, they *should* already know what's what.